otcextensions/tests/functional/sdk/auto_scaling/v1/test_instance.py

```diff
@@ -149,13 +149,14 @@ def _create_as_group(self, as_config_id, router_id, network_id):
         }
         as_group = self.conn.auto_scaling.create_group(**group_attrs)
         self.conn.auto_scaling.resume_group(as_group)
-        return as_group
+        return self.conn.auto_scaling.wait_for_group(as_group)
 
     def _delete_as_group(self, as_group):
         self.conn.auto_scaling.pause_group(as_group)
-        return self.conn.auto_scaling.delete_group(
+        self.conn.auto_scaling.delete_group(
             group=as_group
         )
+        return self.conn.auto_scaling.wait_for_delete_group(as_group)
```

Comment:
"return __delete__" is the same as "return None" is the same as "__delete__"

Reply:
"return" was returned